src/frontend/admin/src/tests/course-run/course-run.test.e2e.ts

```diff
@@ -196,6 +197,7 @@ test.describe("Course run form", () => {
       .click();
     await page.getByRole("gridcell", { name: "27" }).nth(1).click();
     await page.getByRole("button", { name: "OK" }).click();
+    await page.waitForTimeout(20);
```

Comment:
The timeout looks a bit arbitrary nope? Why the next await isn't adequate?

Reply:
Instead of this arbitrary timeout, could we not use `page.waitForEvent` (`transitionend`, 'animationend' ?)

Reply:
I agree